doc/pages/keys.asciidoc

```diff
@@ -552,7 +552,8 @@ to skim through the jump list using:
     jump backward
 
 *<c-s>*::
-    save selections
+    create a jump step, but also save the current selection to have it be
+    restored when the step is subsequently cycled through
```

Comment:
> but also save the current selection to have it be restored

I think that's also the behavior for other jump list entries.. Kakoune might fail restore all selections if the buffer has been modified but that's also independent of `<c-s>`, right?